### DISCUSSION TITLE
cannot find Settings

### DISCUSSION THREAD

**Discussion by githubzoidal** [sumatrapdfreader/sumatrapdf#4326] [Q&A]
The Manual, while it has a lot of references to using Settings, doesn't seem to have any help in _finding the Settings menu_. The Forum likewise has no help I see. This implies that it's **obvious**, but none of the buttons, including the home button, has a link to Settings. If this was some new browser or email client I'd look in the customize menu to enable showing the old File menu. But not here. Perhaps there's a keyboard combo, ctrl-alt-s or something?

**Comment by GitHubRulesOK** [comment]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/abc8120c-d118-4b30-85f0-485035c70fd8)

**In English it would be Alt + S + A**
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/87b919d0-b09b-438e-b203-297876f355e7)

**Reply by githubzoidal** [reply]
Thank you. The first snap shows exactly what I meant by "old File menu". If it were showing on my Sumatra I wouldn't have asked. It's not and that's why. The Alt+S key is sort of what I was looking for, then I can go into Advanced and there's a sort of Config txt. But in there I don't see how, for instance, to get light text on a dark background. The Dark Theme only applies to the general UI, not to documents.

**Reply by GitHubRulesOK** [reply]
Hmm
 recent versions have changed some options but should always be to toggle toolbar (F8) / **menu (F9)**

Reverse colours is "Invert" and on several versions simply `I` key (or else Shifted `I`)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6066eb00-7ec6-4e05-b5d5-8f5786cffc1f)


Depending on version the older manual foreground text and background paper (with auto inversion if you exceed mid range) should work in current pre-release
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/ad3cf15b-01d6-4212-b3f1-c92209839f2a)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/38df07e0-7da9-486a-a028-bfc4d431242c)

**Reply by githubzoidal** [reply]
Thanks again. For me F8 toggles the line that has Page and Find but not the File menu. F9 does nothing. OTOH, the I key does invert the content to documents.

**Reply by GitHubRulesOK** [reply]
if F9 is not showing menu then you must have tabs active ?
so then its the pull down menu which as shown depends on language but for English is ALT+S for settings then A for advanced

**Reply by githubzoidal** [reply]
Not sure about Tabs. I have what I'd call 2 tabs. First is Home, which shows several thumbnails of Frequently Read. Second is a single ebook. I'd still like to see the File menu.

**Reply by GitHubRulesOK** [reply]
when you  press **Left Alt key** or click the menu symbol(s) the dropdowns will appear ![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/c6737c26-a547-4df4-ab1e-5c65f4022b43)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/549c69a9-4da3-4800-8cf3-787ea16923e9)

**Comment by kjk** [comment]
To summarize, here are ways to get advanced options / settings:
* press left `Alt` to trigger menu, use `Settings` and `Advanced options...`
* the same using keyboard: `Alt`, `s`, `a`. This is for English languages and uses keyboard accelerators for menu items (the underlined letter in menu string)
* `Ctrl + K` for command palette, type all or part of `advanced options` or `advanced settings`, Enter to invoke the command
